instrumentation/akka-actor-2.5/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/akkaactor/AkkaActorInstrumentationModule.java

```diff
@@ -34,6 +38,7 @@ public AkkaActorInstrumentationModule() {
     map.put(Runnable.class.getName(), State.class.getName());
     map.put(Callable.class.getName(), State.class.getName());
     map.put(AkkaForkJoinTaskInstrumentation.TASK_CLASS_NAME, State.class.getName());
+    map.put("akka.dispatch.Envelope", State.class.getName());
```

Comment:
Why not "Envelope.class.getName()" ? AFAIK the class is available for reference at this point. Not that it changes a lot ;)

Reply:
Actually the class isn't available. `InstrumentationModule` instances are loaded by `AgentClassLoader` so they can only see what jdk has and the things that are bundled inside agent.

Reply:
That explains a lot ;)